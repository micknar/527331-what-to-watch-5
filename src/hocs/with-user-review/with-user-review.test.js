import React from "react";
import renderer from "react-test-renderer";
import withUserReview from "./with-user-review";
import childrenProp from "../../const/children-prop";

const MockComponent = (props) => {
  const {children} = props;

  return (
    <>
      {children}
    </>
  );
};

MockComponent.propTypes = {
  children: childrenProp.isRequired,
};

const MockComponentWrapped = withUserReview(MockComponent);

it(`withUserReview is rendered correctly`, () => {
  const tree = renderer
    .create((
      <MockComponentWrapped
        filmId={1}
        backgroundColor={`#4aa2e5`}
      >
        <React.Fragment />
      </MockComponentWrapped>
    ))
    .toJSON();

  expect(tree).toMatchSnapshot();
});
