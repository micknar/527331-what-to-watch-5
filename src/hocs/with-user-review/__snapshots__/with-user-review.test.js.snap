// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withUserReview is rendered correctly 1`] = `null`;
